### PR TITLE
Clarifications

### PR DESCRIPTION
**Pull request by cloveras**
Improvements, hopefully.

### CODE REVIEW

File: vipps-login-api-faq.md [hunk 1]
```diff
@@ -120,24 +132,30 @@ Please check:
 * Have you activated Vipps Login and set up a redirect URI? See:
   [How can I activate and set up Vipps Login?](#how-can-i-activate-and-set-up-vipps-login)
 * Have you double checked that the `redirect_uri` used in the API call is
-  _exactly_ the same as the one specified on [portal.vipps.no](https://portal.vipps.no)?
-  Be extra careful with trailing `/` and URL-encoded entities.
+  _exactly_ the same as the one specified on
+  [portal.vipps.no](https://portal.vipps.no)?
+* Pay extra attention to whether the URI used in the3 API request has a
```
Comment (thormarius) [inline vipps-login-api-faq.md:137]: typo in `the3`

File: vipps-login-api-faq.md [hunk 2]
```diff
@@ -40,23 +40,35 @@ the page which the user is redirected to after finishing a login. See:
...
 ![You will find the Setup Vipps Login option in the same place as you find your `client_id` and `client_secret`](images/portal_setup_login.png)
 
-First you activate Vipps Login:
+First you activate Vipps Login by clicking the "Activate Vipps Login" button:
+
 ![First you activate Vipps Login](images/portal_setup.jpeg)
 
 Then you can add the redirect URIs you need:
+1. Enter the URI. It must be _exactly_ the same URI that you use when making
+   API requests. Remember if t he actual URI specified with `redirect_uri` in
```
Comment (thormarius) [inline vipps-login-api-faq.md:57]: typo `t he`

File: vipps-login-api-faq.md [hunk 3]
```diff
@@ -40,23 +40,35 @@ the page which the user is redirected to after finishing a login. See:
...
+   the API requests will have a trailing slash `/` or not.
```
Comment (thormarius) [inline vipps-login-api-faq.md:58]: there is no API request involved here, it's a url where the end user is redirected

File: vipps-login-api-faq.md [hunk 4]
```diff
@@ -40,23 +40,35 @@ the page which the user is redirected to after finishing a login. See:
...
+2. Click "Add URI" for the one or more URIs you are adding.
+3. Click "Save" to save all the URIs you have added.   
+
 ![Then you can add the redirect URIs you need](images/portal_direct_uris.jpeg)
 
 ## What are the requirements for redirect URIs?
 
-We validate redirect URIs against a whitelist of pre-approved URIs. The URIs
-must be registered by the merchant on [portal.vipps.no](https://portal.vipps.no).
+We validate redirect URIs used in API request against a whitelist of
```
Comment (thormarius) [inline vipps-login-api-faq.md:66]: again, this is an `/auth`request where the end user is redirected, not an API request

File: vipps-login-api-faq.md (hunk 2 above)
Comment (cloveras) [inline vipps-login-api-faq.md:57]:
```suggestion
   API requests. Remember if the actual URI specified with `redirect_uri` in
```

File: vipps-login-api-faq.md (hunk 1 above)
Comment (cloveras) [inline vipps-login-api-faq.md:137]:
```suggestion
* Pay extra attention to whether the URI used in the API request has a
```

**Pull request merged**